lib/Runtime/Types/DictionaryTypeHandler.cpp

```diff
@@ -2909,6 +2909,41 @@ namespace Js
     }
 #endif
 
+#if DBG_DUMP
+	template<typename T> void DictionaryTypeHandlerBase<T>::Dump(unsigned indent) const {
+        const auto padding(_u(""));
+        const unsigned fieldIndent(indent + 2);
+        const unsigned mapLabelIndent(indent + 4);
+        const unsigned mapValueIndent(indent + 6);
+
+        Output::Print(_u("%*sDictionaryTypeHandlerBase (0x%p):\n"), indent, padding, this);
+        if (this->propertyMap == nullptr)
+        {
+            Output::Print(_u("%*spropertyMap: null\n"), fieldIndent, padding);
```

Comment:
nit: `<null>` instead? to be similar with others?

Reply:
Fixed